src/Plugin/Field/FieldFormatter/GoogleFormFormatter.php

```diff
@@ -38,11 +54,18 @@ public static function isApplicable(FieldDefinitionInterface $field_definition)
    * {@inheritDoc}
    */
   public function viewElements(FieldItemListInterface $items, $langcode) {
+
     $elements = [];
 
     /** @var \Drupal\Core\Entity\Plugin\DataType\EntityAdapter $parent */
     $parent = $items->getParent();
     $iframe_title = $parent->getEntity()->label();
+    $fields = $parent->getEntity()->getFields();
+
+    if (!empty($fields[$this->iframeHeightField])) {
+      $iframe_height = !empty($fields[$this->iframeHeightField]->getValue()[0]['value']) ?
+        $fields[$this->iframeHeightField]->getValue()[0]['value'] : '600';
+    }
```

Comment:
This is difficult to read. Can you make this more readable.

Reply:
I think my updated version reads a little better.